kernel/fork.c

```diff
@@ -940,6 +940,10 @@ static struct task_struct *dup_task_struct(struct task_struct *orig, int node)
 
 #ifdef CONFIG_MEMCG
 	tsk->active_memcg = NULL;
+#endif
+#ifdef CONFIG_DF_DETECTION
+        tsk->dfetch_addresses = NULL;
```

Comment:
Please mind the indentation (looks like you have 8 spaces on this line and a tab on the following line)